hail_search/hail_search_query.py

```diff
@@ -59,6 +62,15 @@ class BaseHailTableQuery(object):
         XPOS: lambda r: [r.xpos],
     }
 
+    @classmethod
+    def load_globals(cls):
+        loaded_globals = {}
+        for genome_version in cls.GENOME_VERSIONS:
+            ht_path = cls._get_generic_table_path(genome_version, 'annotations.ht')
+            globals_ht = hl.read_table(ht_path).head(0).select()
```

Comment:
can we just access the globals themselves w/ `globals_struct = hl.read_table(ht_path).globals`?

Reply:
🤦 yes